src/test/scala/com/cognite/spark/datasource/TimeSeriesUpsertTest.scala

```diff
@@ -91,6 +92,191 @@ class TimeSeriesUpsertTest extends FlatSpec with SparkTest {
     assert(initialDescriptionsAfterUpsert.count == 0)
   }
 
+  it should "support inserts, updates and upserts in savemode" taggedAs WriteTest in {
```

Comment:
I think we should split those into separate tests, it would make it easier to pinpoint what went wrong if one case fails.